src/index.spec.js

```diff
@@ -37,6 +49,53 @@ describe('Dropzone', () => {
       expect(dropzone.html()).toMatchSnapshot();
     });
 
+    function collectEventListenerCalls(calls) {
+      return calls.reduce((acc, [eventName, ...rest]) => {
+        acc[eventName] = rest; // eslint-disable-line no-param-reassign
+        return acc;
+      }, {});
+    }
+
+    it('prevents stray drops from taking over the browser window by default', () => {
+      const dropzone = mount(<Dropzone><p>Content</p></Dropzone>);
+      expect(dropzone.html()).toMatchSnapshot();
+      expect(document.addEventListener.callCount).toEqual(2);
+      const addCalls = collectEventListenerCalls(document.addEventListener.args);
+
+      const event = { preventDefault: spy() };
```

Comment:
The problem is that there's no way to fire an event on `document`. So if we didn't manually exercise it here, we wouldn't get test coverage. And it's useful to make sure that we don't call `e.preventDefault()` in the case where the drag event is meant for one of our dropzones - because if we did call `e.preventDefault()` in that case, the dropzone would not get the event.

```
1: document onDrop() <-- stops propagation -->
2: dropzone instance onDrop() <-- doesn't get event now -->
```